mycrt-backend/src/server.py

```diff
@@ -107,21 +108,64 @@ def get_rds_instances():
 
     return jsonify({'status': response['ResponseMetaData']['HTTPStatusCode'], 'error': response['Error']['Code']})
 
-@app.route('/login-test', methods=['PUT'])
+@login_manager.request_loader
+def load_user_from_request(request):
+    auth = request.authorization
+    if auth:
+        username = auth.username
+        password = auth.password
+        user = user_datastore.find_user(username=username)
+        if user is not None and verify_password(password, user.password):
+            return user
+    return None
+
+@app.route('/login', methods=['POST'])
 def login():
-    user_repository.register_user(username="test-user", password="test123", email='test@test.com', access_key="test_access_key", secret_key="test_secret_key")
-
+    user = current_user
+    if user is not None:
+        login_user(user)
+        return jsonify({"status" : "success",
+            "user" : {
+                "username" : user.username,
+                "password" : user.password,
+                "access_key" : user.access_key,
+                "secret_key" : user.secret_key,
+                "email" : user.email
+                }
+            }), 200
+    else:
+        return jsonify({"status" : "fail",

```

Comment:
We are sending generic failures as 400, so `return jsonify({"status" : 400,` is what you're looking for
(same above for success)

Reply:
The code after the JSON object is the response code. 
The whole login and home page is kinda hackey because I can't stop the redirect, but I just wanted to get something that works so Audrey could continue on her end. I plan on removing flask-security so it resolves some of these issues.